src/components/WorkspaceAdd.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ns from '../config/css-ns';
+import ManifestForm from '../containers/ManifestForm';
+import ManifestListItem from '../containers/ManifestListItem';
+
+/**
+ * Represents a work area that contains any number of windows
```

Comment:
```suggestion
 * An area for managing manifests and adding them to workspace
```
I think this was copied over from the `Workspace` component.